core/templates/pages/exploration-player-page/templates/lesson-information-card-modal.component.html

```diff
@@ -104,7 +104,7 @@ <h4 tabindex="0" [ngClass]="(checkpointCount === 1) ? 'd-none' : 'progress-bar-l
             </span>
             {{ lastUpdatedString }}
           </li>
-          <ul tabindex="0" class="oppia-info-card-exploration-contributors-profile row center row-item">
+          <ul tabindex="0" class="oppia-info-card-exploration-contributors-profile row center row-item" (click)="closeModal()">
```

Comment:
Hm, do we know why this actually happens? i.e. why is it that navigating away from the page still keeps stale elements on the page.

I think it is important to understand this to prevent this from happening in the future -- we may have modals with links to other pages within the app and might result in similar issues.

Reply:
@kevintab95 , It is likely that modal is still being referenced somewhere even after it is closed when we navigate to another page